packages/teams-js/src/public/navigation.ts

```diff
@@ -46,7 +46,7 @@ export function navigateToTab(tabInstance: TabInstance, onComplete?: onCompleteH
 
 /**
  * @deprecated
- * As of 2.0.0, please use {@link pages.navigateCrossDomain pages.navigateCrossDomain(url: string): Promise\<void\>} instead.
+ * As of 2.0.0, this API is no longer needed. Please remove any calls to it.
```

Comment:
>As of 2.0.0, this API is no longer needed. Please remove any calls to it. [](http://example.com/codeflow?start=3&length=73)

What should developers who read this do instead?

Can you delete the text below this since developers shouldn't use it anymore?